src/plugins/variational_constraints/variational_constraints_engine.jl

```diff
@@ -287,9 +287,9 @@ specific_submodel_constraints(c::Constraints) = c.specific_submodel_constraints
 
 function Constraints()
     return Constraints(
-        Vector{FactorizationConstraint}[],
-        Vector{PosteriorFormConstraint}[],
-        Vector{MessageFormConstraint}[],
+        Vector{FactorizationConstraint}(),
```

Comment:
This actually a funny one @wouterwln , not sure who of us made this mistake, but it basically created a `vector of vector of constraints` instead of just `vector of constraints`. It somehow worked though (probs some weird convert mechanics)